reference/v8.json

```diff
@@ -823,13 +897,19 @@
       "function": "piecewise-constant",
       "zoom-function": true,
       "property-function": true,
-      "values": [
-        "map",
-        "viewport",
-        "auto"
-      ],
+      "values": {
+        "map": {
+            "doc": "When `symbol-placement` is set to `point`, produces glyphs whose x-axes are aligned east-west. When set to `line`, produces glyphs whose x-axes are aligned with the line."
```

Comment:
> Point: Aligns text east-west. 

👍 

> Line: Places text along corresponding line geometry.

There's an important nuance that this doesn't convey; it's the difference between:

```
    ╱
   ↗
  ╱
 ↗
╱
```

and

```
    ╱
   →
  ╱
 →
╱
```